frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx

```diff
@@ -14,37 +18,108 @@ import { SettingTextInput } from "../widgets/SettingTextInput";
 import { useMergeSetting } from "./hooks";
 import type { AdminSettingComponentProps } from "./types";
 
-const SDK_ORIGINS_SETTING = {
-  key: "embedding-app-origins-sdk",
-  placeholder: "https://*.example.com",
-  display_name: t`Cross-Origin Resource Sharing (CORS)`,
-  description: jt`Enter the origins for the websites or web apps where you want to allow SDK embedding, separated by a space. Here are the ${(
-    <ExternalLink
-      key="specs"
-      href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors"
-    >
-      {t`exact specifications`}
-    </ExternalLink>
-  )} for what can be entered.`,
-} as const;
-
 export function EmbeddingSdkSettings({
   updateSetting,
 }: AdminSettingComponentProps) {
-  const sdkOriginsSetting = useMergeSetting(SDK_ORIGINS_SETTING);
-
+  const isEE = PLUGIN_EMBEDDING_SDK.isEnabled();
```

Comment:
rename, since I'm using this flag more than just to check if `sdkOriginsSetting` is editable.

Reply:
I find that line a bit confusing:
- i believe we usually use `isEE` to mean `ee build`
- we call a `isEnabled` function, which i read as "is the setting enabled"

What about something like this?
```suggestion
  const hasEmbeddingSdkFeature = PLUGIN_EMBEDDING_SDK.hasEmbeddingSdkFeature();
```

Reply:
nit: I'm wondering if we even need the plugin

Reply:
plugin is needed when we need to call `hasPremiumFeature` which we do in this case.

Reply:
regarding the use of `isEE` I'm following what we've been doing
https://github.com/metabase/metabase/blob/e11b3a771717af473fbb6310035df6597cdeb1ee/enterprise/frontend/src/metabase-enterprise/embedding/index.js#L18-L19

https://github.com/metabase/metabase/blob/daadbe1bcc7ba4329bf2e6d884bc6e625de9c703/frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings.tsx#L24

But I'm up for a name change. We should also make the embedding counterpart look similar as well.

Reply:
Ah, it seems like I'm on the git blame of the first use of `const isEE = PLUGIN...` 😅